files/default/sql/ddl/2.2.0__initial_tables.sql

```diff
@@ -1965,6 +1966,20 @@ CREATE TABLE `remote_group_project_mapping` (
   CONSTRAINT `fk_remote_group_project_mapping_1` FOREIGN KEY (`project`) REFERENCES `project` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION
 ) ENGINE=ndbcluster DEFAULT CHARSET=latin1 COLLATE=latin1_general_cs;
 
+CREATE TABLE  IF NOT EXISTS `feature_group_validation` (
```

Comment:
The changes are really hard to follow due to the indentation, can you fix it first?

Reply:
fixed